bundles/org.smarthomej.binding.telenot/src/main/java/org/smarthomej/binding/telenot/internal/handler/TelenotBridgeHandler.java

```diff
@@ -298,6 +298,16 @@ private void readerThread() {
                             TelenotThingHandler.readyToSendData.set(true);
                             logger.trace("Ready to send data");
                             break;
+                        case NOT_USED_CONTACT:
+                            logger.debug("Received {} MsgType | hexString: {}", msgType, message);
+                            if (!usedInputContact.isEmpty()) {
+                                logger.warn("Contact {} not used", usedInputContact.get(0));
```

Comment:
Is this an error state or is this expected? If it's not an error, please reduce to INFO.

Reply:
It is expected. I changed this to INFO